manifests/services/postgresql/keepalived.pp

```diff
@@ -1,17 +1,28 @@
 # Configuring keepalived for the postgres IP
 class profile::services::postgresql::keepalived {
-  $vrrp_password = hiera('profile::keepalived::vrrp_password')
+  $vrrp_password = lookup('profile::keepalived::vrrp_password', String)
 
-  $v4ip = hiera('profile::postgres::ipv4')
-  $v4id = hiera('profile::postgres::ipv4::id')
-  $v4pri = hiera('profile::postgres::ipv4::priority')
-  $v6ip = hiera('profile::postgres::ipv6', false)
-  $v6id = hiera('profile::postgres::ipv6::id', false)
-  $v6pri = hiera('profile::postgres::ipv6::priority', false)
-
-  $management_if = hiera('profile::interfaces::management')
+  $v4ip = lookup('profile::postgres::ipv4', Stdlib::IP::Address::V4)
+  $v4id = lookup('profile::postgres::ipv4::id', Integer)
+  $v4pri = lookup('profile::postgres::ipv4::priority', Integer)
+  $v6ip = lookup('profile::postgres::ipv6', {
+    'value_type'    => Variant[Stdlib::IP::Address::V6, Boolean],
+    'default_value' => false
+    })
```

Comment:
Indentering: }) bør kanskje flyttes to hakk til venstre? 4 steder i denne fila :)

Det er også fint med komma på L10, L14, L18 og L24